client/my-sites/earn/memberships/products.jsx

```diff
@@ -322,6 +348,112 @@ class MembershipsProductsSection extends Component {
 						) }
 					</FormToggle>
 				</FormFieldset>
+			</>
+		);
+	}
+
+	renderEmailTab() {
+		const { translate } = this.props;
+		const { editedPostsEmail, editedCustomConfirmationMessage } = this.state;
+		return (
+			<>
+				<FormFieldset>
+					<h6 className="memberships__dialog-form-header">{ translate( 'Posts via email' ) }</h6>
+					<p>
+						{ translate(
+							'Allow members of this recurring payment plan to opt into receiving new posts via email.'
+						) }{ ' ' }
+						<InlineSupportLink
+							supportPostId="154624"
+							supportLink="https://wordpress.com/support/wordpress-editor/blocks/recurring-payments-button/" // TODO: Link to specific section once article is update
+							showIcon={ false }
+							text={ translate( 'Learn more.' ) }
+						/>
+					</p>
+					<FormToggle
+						onChange={ ( newValue ) => this.setState( { editedPostsEmail: newValue } ) }
+						checked={ editedPostsEmail }
+					>
+						{ this.props.translate(
+							'Email newly published posts to members of this recurring payment plan who have opted in.'
+						) }
+					</FormToggle>
+				</FormFieldset>
+				<FormFieldset>
+					<h6 className="memberships__dialog-form-header">
+						{ translate( 'Custom confirmation message' ) }
+					</h6>
+					<p>
+						{ translate(
+							'Add a custom message to the confirmation email that is sent out for this recurring payment plan.'
+						) }
+					</p>
+					<FormTextArea
+						value={ editedCustomConfirmationMessage }
+						onChange={ ( event ) =>
+							this.setState( { editedCustomConfirmationMessage: event.target.value } )
+						}
+					/>
+				</FormFieldset>
+			</>
+		);
+	}
+
+	renderEditDialog() {
+		return (
+			<Dialog
+				className="memberships__dialog"
+				isVisible={ this.state.showDialog }
+				onClose={ this.onCloseDialog }
+				buttons={ [
+					{
+						label: this.props.translate( 'Cancel' ),
+						action: 'cancel',
+					},
+					{
+						label: this.props.translate( 'Save' ),
+						action: 'submit',
+						disabled: ! this.isFormValid(),
+					},
+				] }
+			>
+				<FormSectionHeading>
+					{ this.state.editedProductId && this.props.translate( 'Edit Recurring Payment plan' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Add New Recurring Payment plan' )` *ES Score: 6*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/recurring-payments-email-settings)